web/Web/web/core/session_utils.py

```diff
@@ -44,14 +47,50 @@ def submit_new_session_form(request):
             topic=form.instance,
             max_number_of_judgments=max_number_of_judgments,
             strategy=strategy,
-            show_full_document_content=show_full_document_content
+            show_full_document_content=show_full_document_content,
         )
         messages.add_message(request,
                              messages.SUCCESS,
                              success_message)
         request.user.current_session = session
         request.user.save()
 
+        try:
+            if 'topic-judgments_file' in request.FILES:
+                judgments_file = TextIOWrapper(request.FILES['topic-judgments_file'].file, encoding='utf-8')
+                judgments = [s.strip().split(',') for s in judgments_file.readlines() if s]
+                judgments_dict = dict()
+                for j in judgments:
+                    doc = DocEngine.get_document_or_none(j[0])
+                    if doc:
+                        try:
+                            doc_title = doc['title']
+                            judgments_dict[j[0]] = (doc_title, Judgment.JudgingChoices(int(j[1])))
+                        except ValueError as e:
+                            pass  # judgements with invalid relevance scores are ignored
+
+
+                for doc_id, (doc_title, relevance) in judgments_dict.items():
+                    Judgment.objects.create(
+                        user=request.user,
+                        session=session,
+                        doc_id=doc_id,
+                        doc_title=doc_title,
+                        relevance=relevance,
+                        source="seed"
```

Comment:
The `source` of a judgment might change later if the user decides to rejudge it somewhere else.
To keep track of things, let's add  

```
historyVerbose = [{
    "username": self.request.user.username,
    "source": "seed",
    "judged": true,
    "relevance": rel
}]
```